.changeset/spotty-beers-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
+---
+
+Add DataTable, Table to drafts entrypoint
```

Comment:
I am wondering when will be the best time to start exporting new components in `experimental` bundle instead of `drafts`? According to [the ADR](https://github.com/primer/react/blob/main/contributor-docs/adrs/adr-010-drafts-are-experimental.md)

Reply:
Great point, reached out about this over in Slack to see where that work is. For this PR I'll add it to `drafts` and will add it to `experimental` once that gets created 👀